src/unstract/sdk/exceptions.py

```diff
@@ -12,5 +12,6 @@ def __str__(self) -> str:
 
 class IndexingError(SdkError):
     def __init__(self, message: str = ""):
-        prefix = "Error with indexing. "
-        super().__init__(prefix + message)
```

Comment:
@hari-kuriakose this prefix is needed as I use it to append context as we bubble up the message [here](https://github.com/Zipstack/unstract-sdk/blob/49635b1557f0abd72efea382611d5249b37a7ad6/src/unstract/sdk/index.py#L235).
![image](https://github.com/Zipstack/unstract-sdk/assets/117059509/cc526e10-4028-4705-8c01-1c22327d62ce)

Otherwise we wouldn't know where the error is coming from

Reply:
@chandrasekharan-zipstack We do not need "Error with indexing" prefix when the type of exception itself is `IndexingError`. Downstream should handle this accordingly.

Reply:
@hari-kuriakose in that case, from the backend we would be prefixing `Error with indexing. ` for all `IndexingError` exceptions.
That would also imply that your particular message would be rendered as 
`Error with indexing. Index not found. Please check vector db adapter settings.`

I think this would work - we could review the error messages for cases which sound redundant (if any) later